corehq/apps/cleanup/models.py

```diff
@@ -10,3 +10,17 @@ class Meta:
         constraints = [
             models.UniqueConstraint(fields=['doc_id', 'doc_type'], name='deletedcouchdoc_unique_id_and_type')
         ]
+
+
+class DeletedSQLDoc(models.Model):
+    doc_id = models.CharField(max_length=126)
+    doc_type = models.CharField(max_length=126)
```

Comment:
Is long enough for all SQL models, which should use `app_name.ModelName` format?